shadan/index.html

```diff
@@ -0,0 +1,72 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <div class="flex-side">
+      <div class="flex-side1">
+          <div class="text-blog">
+    <h3><strogn>MY BLOG</strogn></h3>
+    <p>Where my ideas manifest <br> themselves verbally!</p>
+    <p><a href="#">HOME</a><a href="#">PROJECTS</a></p>
+  </div>
+</div>
+
+  <div class="flex-side2">
+
+    <span class="grey-c">PINNED POST</span>
+    <hr class="line">
+    <h2>First blog post</h2>
+    <img src="imgs/computer.jpg" class="blog-img">
+    <p>By<a href="#"> Not Salah Al-Dhafteri</a> under <span class="green">Css</span> <span class="blue">Javascript</span></p>
+    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+      book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+      more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. </p>
+
+    <span class="grey-c recent">RECENT POSTS</span>
+    <hr class="line">
+    <h2>How I go about my living day </h2>
+    <img src="imgs/computer.jpg" class="blog-img">
+    <p>By<a href="#"> Not Salah Al-Dhafteri</a> under <span class="red">Javascript</span></p>
+    <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+      book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and
+      more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. </p>
+
+    <h2 class="recent">Photos from that time when I never visited these places</h2>
+    <img src="imgs/computer.jpg" class="blog-img">
+    <p>By<a href="#"> Not Salah Al-Dhafteri</a> under <span class="grey">Not visited places</span>
+      <div>
+
+        <img class="img-bottom" src="imgs/Beautiful2.jpg">
+        <img class="img-bottom" src="imgs/beautiful3.jpg">
+
+        <div>
+          <table>
+            <thead>
+              <tr>
+                <td width="100px">About</td>
+                <td width="100px">Twitter</td>
+                <td>Github</td>
+              </tr>
+            </thead>
+          </table>
+        </div>
+
+      </div>
+
+  </div>
+  <div class="flex3">
+
```

Comment:
I'm not sure why there is an empty div here?

Reply:
We considered the third part of the page is blank . 
First part contains blog 
Second part contains posts
Third part to create a blank part after second part